src/js/basic-pubsub/index.js

```diff
@@ -79,15 +85,15 @@ const eventService = {
     let args = Array.prototype.slice.call(arguments)
     let topicId = args.shift()
 
-    let topic = _.find(this.topics, {id: topicId})
+    let topic = this.getTopic(topicId);
```

Comment:
derp, why didn't I write it this way the first time

Reply:
heat of the moment, I'm sure :D